src/cmd/input.go

```diff
@@ -28,6 +29,16 @@ func readInputConfig() {
 	viper.ReadInConfig()
 }
 
+func readResourceInput[T any]() (*T, error) {
+	readInputConfig()
+	var evt *T
+	viper.Unmarshal(&evt)
```

Comment:
We shouldn't use viper here - it conflates with the options vars and could cause unexpected problems if a CLI flag overlaps with an input type field in name.  We should use `yaml.Unmarshal` by reading the input file using `os.ReadFile`.

This has been on my backburner to fix